src/NLog.Web.AspNetCore/NLog.Web.AspNetCore.csproj

```diff
@@ -75,14 +75,15 @@ NLog 5 release post: https://nlog-project.org/2021/08/25/nlog-5-0-preview1-ready
     <DefineConstants>$(DefineConstants);ASP_NET_CORE;ASP_NET_CORE3</DefineConstants>
   </PropertyGroup>
   <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Http.Extensions" Version="2.1.21" />
     <PackageReference Include="NLog.Extensions.Logging" Version="5.0.0" />
+    <PackageReference Include="System.Text.Encodings.Web" Version="4.5.1" />
```

Comment:
Please move this nuget-dependency back to its original location.

Reply:
ok I will undo it